src/intern/intern.json

```diff
@@ -135,6 +135,22 @@
           "platform": "WIN10"
         }
       ]
+    },
+    "headless": {
+      "extends": "built",
+      "tunnelOptions": {
+        "drivers": [
+          { "name": "chrome", "version": "76.0.3809.68" }
+        ]
+      },
+      "environments": [
```

Comment:
@nka089 Could we please also add `{ "browserName": "node" }` as an environment, similar to the `local` configuration, thanks 😄